seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobExecutorServiceImpl.java

```diff
@@ -63,6 +65,14 @@
 @Slf4j
 @Service
 public class JobExecutorServiceImpl implements IJobExecutorService {
+    public static final String DIR_PROFILE = "profile";
```

Comment:
can you make these three constants private.

Reply:
done